source/dofs/dof_handler_policy.cc

```diff
@@ -4235,130 +4225,11 @@ namespace internal
         template <int dim>
         struct CellDataTransferBuffer
         {
-          std::vector<unsigned int> tree_indices;
-          std::vector<typename dealii::internal::p4est::types<dim>::quadrant>
+          std::vector<
```

Comment:
What is the use of this struct now? Either we should put some of the logic inside it or just remove it. We don't communicate the whole struct anymore anyways, right?

Reply:
You are right, we could simply remove it. Let me try it (though not immediately).

Reply:
I fixed it now, indeed the code is even cleaner.